src/main/java/application/onlinebookstore/dto/cartitem/CartItemDtoUpdate.java

```diff
@@ -0,0 +1,14 @@
+package application.onlinebookstore.dto.cartitem;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.validation.constraints.DecimalMin;
+import lombok.Data;
+
+@Data
+public class CartItemDtoUpdate {
+    @JsonIgnore

```

Comment:
From my experience it's not the best practice to set something to dtos after their creation, it can cause a lot of undesired side effects, better to remove this field and pass id to the method as a separate param (It's just a suggestion, so you decide what is better for you)